examples/opensora_hpcai/opensora/models/stdit/stdit2.py

```diff
@@ -135,9 +135,10 @@ def construct(
             )
 
         # modulate
-        x_m = t2i_modulate(self.norm1(x), shift_msa, scale_msa)
+        norm1 = self.norm1(x).to(ms.float32)  # t2i_modulate involves parameter normalization, requires high precision
```

Comment:
Original OpenSora just uses bf16. Is it MindSpore specific to use fp32? There is no division involved here, so I think it's safe to use bf16 as well.

Reply:
Additionally, `to(ms.float32)` won't work in PyNative.